cppapi/client/api_util.cpp

```diff
@@ -484,7 +484,7 @@ void ApiUtil::get_asynch_replies()
     }
     catch (CORBA::BAD_INV_ORDER &e)
     {
-        if (e.minor() != omni::BAD_INV_ORDER_RequestNotSentYet)
+        if ((e.minor)() != omni::BAD_INV_ORDER_RequestNotSentYet)
```

Comment:
Do you know why this is needed? Looks like minor is just a function in SystemException:
```cpp
  inline ULong minor() const { return pd_minor; }
  inline void minor(ULong m) { pd_minor = m;    }
```

Reply:
@mliszcz No I have no idea.